src/app/Fake.Core.UserInput/UserInput.fs

```diff
@@ -0,0 +1,63 @@
+namespace Fake.Core
+open System
+
+/// Helpers for capturing user input
+///
+/// ## Sample
+///
+///     UserInput.getUserInput prompt
+[<RequireQualifiedAccess>]
+module UserInput =
+let internal erasePreviousChar () =
```

Comment:
The code if the module needs to be intended to the right in order to get rid of the warning:

C:\projects\fake-6w516\src\app\Fake.Core.UserInput\UserInput.fs(11,1): warning FS0058: Possible incorrect indentation: this token is offside of context started at position (10:1). Try indenting this token further or using standard formatting conventions. [C:\projects\fake-6w516\src\app\Fake.Core.UserInput\Fake.Core.UserInput.fsproj]

Basically just intend from line 11 to line 63